### PR TITLE
Created a 'Packet' object for our database and setup some basic routing

### LINKED ISSUE #3: Get a single network packet from a live connection:
- Interpret the data

### PR DESCRIPTION
**Pull request by seanjhulse**
# Preface

Hey team, I didn't realize this, but we still had a _little_ more to go with our skeleton commit.

Django has two concepts:

1. A project. A project is like a generic structure that includes basic settings and admin-level details
2. An app. An app is your actual web application. You can create an app like `python manage.py startapp NAME-OF-YOUR-APP`

What we created last Thursday was just a project. Not the actual web app. So, when I was starting to do our work, I realized this and I fixed it. I renamed our `sherLOCK` "project" to become `sherlock-project` and I created a webapp called `sherlock` using `python manage.py startapp sherlock`. 

99% of our work will _probably_ live within this `sherlock` app folder.

____

# Actual Work

Most of my actual work is inside `sherlock/urls.py` and `sherlock/views.py`. I created a few example routes in Django and one route that actual does what this ticket is meant to solve. 

`def network_traffic(request, port):` is the route that actually grabs data from the socket. In order for this to work **you must run the Django project as sudo** (eg. `sudo python3 manage.py runserver`). This is probably not going to be a long-term solution since running as sudo is not the smartest thing in the world. But I really wanted to get this ticket across the board since this whole `project vs. app` thing is important for our skeleton

____

# Notes

There are details in the README.md to show you how to create your own routes and an example route inside `sherlock/urls.py` and `sherlock/views.py` so you can see how it works.

### CODE REVIEW

File: sherlock/models.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    dest_ip_address = models.CharField(max_length=39)
+
+    # Port limited to 5 characters
+    port = models.CharField(max_length=5)
+
+    # Data is just an "unlimited" TextField
+    data = models.TextField
+
+    # Should store when this data was published to the database
+    pub_date = models.DateTimeField('date published')
```
Comment (seanjhulse) [inline sherlock/models.py:24]: This `sherlock/models.py` is just an example model for now. I am going to work on this more for my other ticket. Right now it's just sitting here. If you have comments, let me know. I'll save them and address them on that ticket later on.

**Review by gioflores24 — COMMENTED** [review]
I approve of the pull request.

**Comment by casasso83** [comment]
Sounds solid. Currently working on my ticket for a pseudo network data pull from an endpoint.

**Review by casasso83 — COMMENTED** [review]
Sounds solid. Currently working on my ticket for a pseudo network data pull from an endpoint.

**Pull request merged**